src/Rector/ClassMethod/DependsAnnotationWithValueToAttributeRector.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\PHPUnit\Rector\ClassMethod;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\ClassMethod;
+use PHPStan\PhpDocParser\Ast\PhpDoc\GenericTagValueNode;
+use PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTagNode;
+use Rector\BetterPhpDocParser\PhpDocInfo\PhpDocInfo;
+use Rector\BetterPhpDocParser\PhpDocManipulator\PhpDocTagRemover;
+use Rector\Core\Rector\AbstractRector;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+use Rector\PhpAttribute\NodeFactory\PhpAttributeGroupFactory;
+use Rector\PHPUnit\NodeAnalyzer\TestsNodeAnalyzer;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\PHPUnit\Tests\Rector\ClassMethod\DependsAnnotationWithValueToAttributeRector\DependsAnnotationWithValueToAttributeRectorTest
+ */
+final class DependsAnnotationWithValueToAttributeRector extends AbstractRector
+{
+    public function __construct(
+        private readonly TestsNodeAnalyzer $testsNodeAnalyzer,
+        private readonly PhpAttributeGroupFactory $phpAttributeGroupFactory,
+        private readonly PhpDocTagRemover $phpDocTagRemover
+    ) {
+    }
```

Comment:
The dependency injection here seems cause rectify error:

```bash                                                                                
 [ERROR] Could not process                                                      
         "/home/runner/work/rector-phpunit/rector-phpunit/src/Rector/ClassMethod
         /DependsAnnotationWithValueToAttributeRector.php" file, due to:        
         "System error: "Call to undefined method                               
         PhpParser\Internal\TokenStream::haveTagInRange()"                      
         Run Rector with "--debug" option and post the report here: 
https://github.com/rectorphp/rector/issues/new
". On line:
         882                                                                    
```

Reply:
It seems resolved by directly using string in args on `$this->phpAttributeGroupFactory->createFromClassWithItems()` https://github.com/rectorphp/rector-phpunit/pull/133/commits/4d1244f02e5870084ccbb4e1a59a75b6595d8236

which really strange, but works ...